packages/salesforcedx-vscode-core/src/index.ts

```diff
@@ -541,6 +543,11 @@ export async function activate(extensionContext: vscode.ExtensionContext) {
   // thus avoiding the potential errors surfaced when the libs call
   // process.cwd().
   ensureCurrentWorkingDirIsProjectPath(rootWorkspacePath);
+  const platform = process.platform.toLowerCase();
+  const isWindows = platform === 'win32';
+  if (!isWindows) {
```

Comment:
I'm not sure about this 💀 because we have so many users on windows... and we would be 100% skipping the validation for them. Is this something we want? It seems like half a feature